motor/motor_gridfs.py

```diff
@@ -212,15 +207,13 @@ def open(self):
            :class:`~motor.MotorGridOut` now opens itself on demand, calling
            ``open`` explicitly is rarely needed.
         """
-        return self._framework.chain_return_value(self._ensure_file(),
-                                                  self.get_io_loop(),
-                                                  self)
+        await self._ensure_file()
+        return self
```

Comment:
Is this a breaking change? The docstring says "Returns a Future." but now we return an async awaitable.

Reply:
Can you open a ticket for this change? Perhaps we'll think of a better solution later.